Rmarkdown/Multiple_Ascending_Dose_PKPD_ordinal.Rmd

```diff
@@ -96,119 +112,70 @@ Summarize the data in a way that is easy to visualize the general trend of PD ov
 
 ```{r, cache = TRUE, fig.width = 10, fig.height = 4, warning = FALSE, message= FALSE}
-data_to_plot <- list()  
-data_to_plot[[1]] <- my.data[my.data$CMT==2,]
-data_to_plot[[1]]$TRTACT <- factor(data_to_plot[[1]]$TRTACT, levels = rev(levels(data_to_plot[[1]]$TRTACT)))
-
-data_to_plot[[2]] <- my.data[my.data$CMT==5,]
-data_to_plot[[2]]$TRTACT <- factor(data_to_plot[[2]]$TRTACT, levels = rev(levels(data_to_plot[[2]]$TRTACT)))
-data_to_plot[[2]]$Response <- factor(data_to_plot[[2]]$Severity_label, levels = rev(levels(data_to_plot[[2]]$Severity_label)))
-
-data_to_plot[[2]]$DAY <- floor(data_to_plot[[2]]$TIME/24 )
-
-ggs <- list()
-for(iplot in 1:length(data_to_plot)){
-  
-     if(iplot == 1){
-  
-  gg <- ggplot(data = data_to_plot[[iplot]], 
-               aes(x=NOMTIME/24,y=LIDV, color = TRTACT, fill = TRTACT))+theme_bw()
-  gg <- gg + stat_summary(geom="errorbar", 
-                          fun.data=function(y){
-                            data.frame(y=mean(y),
-                                       ymin = mean(y)-qt(0.975,length(y))*sqrt(stats::var(y)/length(y)),
-                                       ymax = mean(y)+qt(0.975,length(y))*sqrt(stats::var(y)/length(y))
-                          )}, size = 1, width = 0)
-  gg <- gg + stat_summary(geom="point", fun.y=mean)
-  gg <- gg + stat_summary(aes(group = TRTACT), geom="line", 
-                          fun.y=mean)+ scale_x_continuous(breaks = seq(-1,8,1))
-      gg <- gg + scale_y_log10() + annotation_logticks(base = 10, sides = "l", color = rgb(0.5,0.5,0.5))
-    gg <- gg + ylab("Concentration (ng/mL)")
-
-    }else{
-      gg <- ggplot(data = data_to_plot[[iplot]], aes(x = factor(DAY), fill = Response)) + theme_bw()
+#PK data
+gg <- ggplot(data = pk_data, 
+               aes(x=NOMTIME,y=LIDV, color = TRTACT_high2low, fill = TRTACT_high2low)) 
+gg <- gg + xgx_stat_ci(conf_level = .95)
+gg <- gg + xgx_annotate_status(status)
+gg <- gg + xgx_scale_x_time_units(units_dataset = time_units_dataset, 
+                                  units_plot    = time_units_plot)
+gg <- gg + guides(color=guide_legend(""),fill=guide_legend(""))
+gg <- gg + xgx_scale_y_log10()
+gg <- gg + labs(y=conc_label)
+gg
+
+#PD data
+gg <- ggplot(data = pd_data, aes(x = factor(DAY), fill = Severity_high2low)) 
 gg <- gg + geom_bar(position = "fill") + scale_y_continuous(labels = scales::percent)
-gg <- gg + ylab("Responder Rate (%)") + guides(fill=guide_legend("")) #+ theme(legend.position = "bottom")
-gg <- gg + facet_grid(TRTACT~.)
+gg <- gg + labs(x=time_label, y=pdresponse_label)
+gg <- gg + facet_grid(TRTACT_high2low~.)
 gg <- gg + scale_fill_brewer(palette=6)
-    }
-  gg <- gg + guides(color=guide_legend(""),fill=guide_legend(""))
-  gg <- gg  + xlab("Time (days)")
-  ggs[[iplot]] <- gg
-}
-
-grid.arrange(ggs[[1]], ggs[[2]], ncol = 2)
+gg <- gg + guides(color=guide_legend(""),fill=guide_legend(""))
+gg <- gg + xgx_annotate_status(status)
+gg
```

Comment:
In the previous version of this page, the PK figure and the PD figure were side-by-side. This was nice because the PD facets were stacked by treatment, and in the same order of the PK profiles. Can we put back the grid.arrange() line to display both figures side by side?

Reply:
![image](https://user-images.githubusercontent.com/44787126/75912473-4605be00-5e1f-11ea-853b-deae253bfcf4.png)